liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/BaseDevTest.java

```diff
@@ -237,4 +247,18 @@ protected static boolean checkLogMessage(int timeout, String message)
       }
       return (waited > timeout);
    }
+
+   protected static boolean verifyLogMessageExists(String message, int timeout)
```

Comment:
This is the same as checkLogMessage except the return values are inverted (which makes more sense) and does not sleep after readFile returns true (which might have been related to a timing issue.  But readFile should have closed the file when it finishes anyways).

Can `checkLogMessage` be deleted, and all references to `checkLogMessage` be replaced with `verifyLogMessageExists`?

Reply:
Yes, I'll make the rest of the changes